src/js/components/AppListComponent.jsx

```diff
@@ -192,34 +192,34 @@ var AppListComponent = React.createClass({
   getGroupsFromApps: function (apps) {
```

Comment:
Shouldn't we be able to remove this method a the list of items already include groups?